lib/x11test.pm

```diff
@@ -81,6 +81,8 @@ sub test_terminal {
     my ($self, $name) = @_;
     mouse_hide(1);
     x11_start_program($name);
+    # GNOME40 sometimes gets its activities mode incorrectly triggered by x11_start_program
```

Comment:
'sometimes' sounds like 'we don't know why, but it happens; it's very clearly happening on boot/login (session start)